hpOneView/connection.py

```diff
@@ -283,7 +283,7 @@ def post_multipart_with_response_handling(self, uri, file_path, baseName):
 
         return None, body
 
-    def post_multipart(self, uri, fields, files, baseName, verbose=False):
+    def post_multipart(self, uri, fields, files, baseName, custom_headers, verbose=False):
```

Comment:
again custom_headers should be added in the end of args list as we discussed and is custom_headers a mandatory param. If not, then better to give default value